### DISCUSSION TITLE
mac os 本地编译显示  unrecognized option '--dev'

### DISCUSSION THREAD

**Discussion by wulizzh** [alibaba/GraphScope#3591] [Q&A] [answered]
![111](https://github.com/alibaba/GraphScope/assets/82989918/bfbc00df-28c9-47ca-a3bc-2a98b7d8a86f)

**Comment by siyuan0322** [comment, marked as answer]
Hi, this is now handled by `gsctl.py` as shown here: https://github.com/alibaba/GraphScope?tab=readme-ov-file#building-on-local

**Reply by JackyYangPassion** [reply]
现在支持Mac 环境直接编译吗？我理解是依赖docker image

**Reply by siyuan0322** [reply]
docker image 最方便，mac 直接编也可以，只是根据mac的不同版本可能需要处理一些编译链/依赖库/环境变量的问题
